src/main/scala/org/scalameter/execution/JvmRunner.scala

```diff
@@ -32,7 +32,10 @@ final class JvmRunner {
   }
 
   private def runJvm(ctx: Context) {
-    val classpath = ctx.goe(Key.classpath, utils.ClassPath.default)
+    val classpath = {
+      val cl = ctx.goe(Key.classpath, utils.ClassPath.default)
```

Comment:
Could we divide the keys into `Key.nonModifiedClasspath` and `Key.classpath` - then `nonModifiedClasspath` would not contain `instrumentedJarPath`, and `classpath` would contain any additions by measurers, executors, etc.

This could reduce the chances of somebody forgetting to add `instrumentedJarPath`. Now we apparently only have `JvmRunner`, but we might have a `ClassLoaderRunner` in the future (which creates a sub-classloader).

We can do this in a separate PR, though.